daemon/main.c

```diff
@@ -941,7 +941,8 @@ int main(int argc, char **argv) {
                             default_rrd_update_every = 1;
                             default_rrd_memory_mode = RRD_MEMORY_MODE_RAM;
                             default_health_enabled = 0;
-                            rrd_init("unittest", NULL);
+                            if(!rrd_init("unittest", NULL))
+                                fatal("rrd_init failed for unittest");
```

Comment:
You should use `fprintf(stderr, ` here since the log service hasn't been initialized yet.